### DISCUSSION TITLE
Package import

### DISCUSSION THREAD

**Discussion by donywan** [serverpod/serverpod#2761] [General]
```
import 'package:collection/collection.dart';
import 'package:serverpod/src/cache/cache_miss_handler.dart';
import 'package:serverpod_serialization/serverpod_serialization.dart';

import 'cache.dart';
```

It is recommended to use relative path import. During use, I found that the import method using `package:` cannot jump and display the class hierarchy in `vscode` through the `dart` plug-in.

**Comment by donywan** [comment]
You can check this issue, https://github.com/Dart-Code/Dart-Code/issues/5280

**Comment by donywan** [comment]
This is because of my local environment.

**Comment by farhansadikgalib** [comment]
Hi brothers, I need some stars for my open source contribution. I would appreciate it if everyone could add a star to this repository.
https://github.com/farhansadikgalib/dropdown_flutter

**Reply by donywan** [reply]
> Hi brothers, I need some stars for my open source contribution. I would appreciate it if everyone could add a star to this repository. https://github.com/farhansadikgalib/dropdown_flutter

OK.
